Update talks.md (#997)
2i2c-org/team-compass: communication/talks.md

## When to give talks

If a talk is virtual and a "standard" talk length, feel free to accept these opportunities to signal-boost our mission and impact, while also growing your own professional reach.

For talks and conferences that take place in-person and require travel that is not reimbursed by the organizers, or if it is particularly complex or important, get approval from a team lead at 2i2c before deciding to attend. Conferences are great opportunities for 2i2c to learn, showcase our abilities, and potentially gather sales leads. 

Below is information to help understand what kinds of opportunities we should look for.

## How to prioritize conference talks and sponsorship opportunities

> We're building a [Playbook for conference attendance and sponsorship](https://docs.google.com/document/d/1jynjVxVPgRlS5_cbI_B6sKMTsZG58dsO9b6RyUaNQnA/edit?tab=t.0), which offers more detail on what actions to take, including goal-setting and prioritization of different opportunities. 

Reviewer: ```suggestion
Use the [Playbook for conference attendance and sponsorship](https://docs.google.com/document/d/1jynjVxVPgRlS5_cbI_B6sKMTsZG58dsO9b6RyUaNQnA/edit?tab=t.0) for ideas of what actions to take, including goal-setting and prioritization of different opportunities. 
```
